priv/tools/internal/select_gc_bucket.erl

```diff
@@ -0,0 +1,199 @@
+%% #!/usr/bin/env escript
+
+%% ---------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% ---------------------------------------------------------------------
+
+-module(select_gc_bucket).
+
+-compile(export_all).
+
+-include_lib("riak_cs/include/riak_cs.hrl").
+
+-record(state,
+        {logger :: pid(),
+         ring_size = 64 :: non_neg_integer(),
```

Comment:
The default value is just to make sure. If lacking, dialyzer will estimate this as `undefined | non_neg_integer()` which is disgusting.